src/openforms/appointments/renderer.py

```diff
@@ -45,8 +45,9 @@ def __str__(self) -> SafeString:
         )
         plugin = get_plugin(plugin=appointment.plugin)
 
+        identifier: str = self.submission.appointment_info.appointment_id
```

Comment:
I see in the `AppointmentInfo` model that the `appointment_id` can also be empty. Are we sure that it always has a value here?

Reply:
yes, for appointments that were succesfully created in JCC/Qmatic/demo, this value is set. It is the return value of `plugin.create_appointment`: https://github.com/open-formulieren/open-forms/blob/3ef4ea6bd4a1d720f67073da2bec37269fd62bc7/src/openforms/appointments/core.py#L55